Source/UI/PaymentForm/View/SecurityCodeViewModel.swift

```diff
@@ -42,7 +42,7 @@ final class SecurityCodeViewModel {
         }
 
         guard let cleanedInput = newInput?.filter({ !$0.isWhitespace }),
-              (Int(cleanedInput) ?? 0 > 0) || cleanedInput.isEmpty,
```

Comment:
Are you sure this is the problem? 

Looks to me like we should just improve the filter to not only remove whitespaces, but any non digit character. Then on the remaining string, check for input length. Why do we need to parse to Int now, or compare with 0?

Reply:
But logically, is that the right logical thing to do?

**Need**
I want to ensure only digits are entered.
I want to ensure the correct number of digits are entered.

At which stage do I need to check if the value of the number created from those digits is comparable to something?